src/ng/compile.js

```diff
@@ -1422,9 +1422,12 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
    * @kind function
    *
    * @description
-   * Call this method to enable/disable strict component bindings check. If enabled, the compiler will enforce that
-   * for all bindings of a component that are not set as optional with `?`, an attribute needs to be provided
-   * on the component's HTML tag.
+   * Call this method to enable / disable the strict component bindings check. If enabled, the
+   * compiler will enforce that for all scope / controller bindings of
+   * a {@link $compileProvider#directive directive} / {$compileProvider#component component}
```

Comment:
`{$compilerProvider#component` --> `{@link $compilerProvider#component`
`of directive / component` --> `of a directive / component` or `of directives / components`